interface/forms/misc_billing_options/new.php

```diff
@@ -34,110 +34,220 @@
+                        <div class="form-group clearfix">
+                            <label for='hospitalization_date_from' class="col-sm-3 form-inline"><?php echo xlt('Box 18. Hospitalization date from');?>:</label>
+                                <?php $hospitalization_date_from = $obj{"hospitalization_date_from"}; ?>
+                                    <input type="text" class='datepicker col-sm-2 ' name='hospitalization_date_from' id='hospitalization_date_from' value='<?php echo attr($hospitalization_date_from); ?>' title='<?php echo xla('yyyy-mm-dd'); ?>'>
+                            <label for='off_work_to'class="col-sm-3 form-inline"><?php echo xlt('Box 18. Hospitalization date to');?>:</label>
+                                <?php $hospitalization_date_to = $obj{"hospitalization_date_to"}; ?>
+                                    <input type="text"  class='datepicker col-sm-2'  name='hospitalization_date_to' id='hospitalization_date_to' value='<?php echo attr($hospitalization_date_to); ?>' title='<?php echo xla('yyyy-mm-dd'); ?>'>
+                        </div>
+                        <div class="form-group">
+                            <label><?php echo xlt('Box 20. Is Outside Lab used?'); ?>:
+                                <input type="checkbox" name="outside_lab" value="1"
+                                <?php
+                                if ($obj['outside_lab'] == "1") {
+                                    echo "checked";
+                                }?>>
+                            </label>
+                            <label><?php echo xlt('Amount Charged'); ?>:
+                                <input type="text" size="7" align='right' name="lab_amount" value="<?php echo attr($obj{"lab_amount"});?>" >
+                            </label>
+                        </div>
+                        <div class="form-group">
+                            <label><?php echo xlt('Box 22. Medicaid Resubmission Code (ICD-10) ');?>:
+                                <input type="text"  name="medicaid_resubmission_code" value="<?php echo attr($obj{"medicaid_resubmission_code"});?>" >
+                            </label>
+                            <label><?php echo xlt(' Medicaid Original Reference No. ');?>:
+                                <input type="text"  name="medicaid_original_reference" value="<?php echo attr($obj{"medicaid_original_reference"});?>" >
+                            </label>
+                        </div>
+                        <div class="form-group">
+                            <label><?php echo xlt('Box 23. Prior Authorization No. ');?>:
+                                <input type="text"  name="prior_auth_number" value="<?php echo attr($obj{"prior_auth_number"});?>" >
+                            </label>
+                        </div>
+                        <div class="form-group">
+                            <label><?php echo xlt('X12 only: Replacement Claim '); ?>:
```

Comment:
remove trailing space from this constant
(btw, just so you know, I am only having you change constants that were previously changed by Stephen; otherwise best to not change them which is why I have not told you to fix them all)